### PR TITLE
Machine learning release notes 10.13

### PR DESCRIPTION
**Pull request by VRAN016**

### CODE REVIEW

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Upgrades:
+  * Machine Learning web application upgraded from Angular version 11 to version 12.
+  * Machine Learning web application upgraded to use Cumulocity Web SDK 1013.0.0.
+  * Nyoka and Onnx microservices upgraded NumPy from version 1.18.1 to version 1.18.5.
+  * Zementis microservice upgraded to use Cumulocity Java Microservice SDK 1011.0.16.
+
+- Bug fixes:
+  - Fixed the issue with `Access is denied` when OAuth is used as the tenant’s authentication scheme.
+  - Fixed the issue with `Resource not found` error message when trying to delete artifacts.
+  - Fixed the issue with artifact download when artifact names contains comma.
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:26]: when artifact names contains comma > either "when artifact name contains comma" or "when artifact names contain comma"

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+* Enable Project Export and Import:\
+Machine Learning Workbench provides a project-based structure for encapsulating data science assets - data, code, resources, and models. This helps Data Scientists and Machine Learning practitioners organize assets relevant to various stages of model lifecycle from data preparation to model building and deployment. To facilitate collaboration and sharing, users can now export the contents of a project as a compressed file and similarly, contents from an exported archive can be uploaded as new project.
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:36]: Data Scientists and Machine Learning practitioners > data scientists and machine learning practitioners

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+* Notebook UI:\
+The Notebook UI of Machine Learning Workbench was a custom implementation created by using Monaco editor. The UI has been upgraded to use off-the-shelf JupyterLab UI components, which is the next-generation user interface for Project Jupyter, offering all the familiar building blocks of the classic Jupyter Notebook (notebook, text editor, rich outputs, etc.) in a flexible and powerful user interface.
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:39]: Monaco editor > the Monaco editor
Project Jupyter > the project Jupyter

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+- Enhancements:
+  - Project improvements:
+    - Enabled resource rename for project artifacts.
+    - Enabled Data validation with type annotations using the Pydantic package for all the MLW APIs.
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:44]: Data > data

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md (hunk 1 above)
Comment (VRAN016) [reply content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md]: Changes incorporated.

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md (hunk 2 above)
Comment (VRAN016) [reply content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md]: Changes incorporated.

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md (hunk 3 above)
Comment (VRAN016) [reply content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md]: Changes incorporated.

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md (hunk 4 above)
Comment (VRAN016) [reply content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md]: Changes incorporated.

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**